play/binary-explorer/js/app.js

```diff
@@ -68,4 +68,27 @@ function translateBinaryString() {
 
     // convert ASCII code to a character and show in result element
     resultEl.innerHTML = String.fromCharCode(asciiCode);
+}
+
+/**
+ * Add letters translated from binary string rows to the message.
+ */
+function addMessage() {
+
+    // create message span if not created before
+    let messageContainer = document.getElementById("message-container");
+    if (document.getElementById("message") == null) {
+        let span = document.createElement("span");
+        span.id = "message";
+        messageContainer.appendChild(span);
+    }
+
+    // insert message to span
```

Comment:
I think that this implementation is very clever, but I had a different expected behavior. I think that a letter or many letters should be able to be added to the message-container, and then their row can be removed. This means that addMessage/addToMessage should be reading the rows when the button is clicked, and appending the letter to the message, not overwriting it.